app/src/test/java/com/example/concordia_campus_guide/ViewModel/MainActivityViewModelTest.java

```diff
@@ -0,0 +1,44 @@
+package com.example.concordia_campus_guide.ViewModel;
+
+import com.example.concordia_campus_guide.Activities.MainActivityViewModel;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.MockitoAnnotations;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class MainActivityViewModelTest {
+    private MainActivityViewModel mainActivityViewModel;
+
+    @Before
+    public void init(){
+        MockitoAnnotations.initMocks(this);
+        mainActivityViewModel = new MainActivityViewModel();
+    }
+
+    @Test
+    public void cleanTimeDisplayLongerThanOneHoursTest(){
+        long date = 15852069;
+        String dateString = mainActivityViewModel.timeUntilMethod(date);
+        String expectedValue ="04 hours and 24 minutes";
+        assertEquals("cleanTimeDisplayLongerThanOneHoursTest(): ", expectedValue, mainActivityViewModel.cleanTimeDisplay(dateString));
```

Comment:
Can you make the message more descriptive? It doesn't have to match the method signature